### ISSUE TITLE
Using up almost all CPU

### ISSUE THREAD

**Issue by medic27** [gcazaciuc/vscode-flow-ide#7]
It increases in CPU usage until all processes are frozen :(

![screen shot 2017-08-10 at 10 30 02 am](https://user-images.githubusercontent.com/6894926/29183282-f54cabcc-7db6-11e7-88b1-08d2bc56a277.png)

**Comment by gcazaciuc** [comment]
I believe this has been raised also with the Flow team. There is nothing we can do on tjhe extension side of things as it seems that 'flow' is the one eating up the memory and not the extension itself. Please file a bug with Flow repo.

**Comment by n1ru4l** [comment]
@gcazaciuc Is this package using the flow-language-server?

Related: https://github.com/flowtype/flow-for-vscode/issues/164

**Comment by gcazaciuc** [comment]
No, it's just calling in directly the flow binary to run `flow check` on the specified file.

**Comment by n1ru4l** [comment]
Okay, I am also experiencing issues of full ram after using this package for a few minutes, I also think this is related to flow itself and not this package. With prior flow versions I did not encounter this problem.
The long term solution on the official package seems to be the usage of the flow-language-server (which is already used by the atom ide integration)

**Comment by dvnrsn** [comment]
Flow still be eating all the cake. Anyone find a solution?
![screen shot 2018-08-22 at 10 48 12](https://user-images.githubusercontent.com/9112811/44477796-eeb34a80-a5f8-11e8-8434-a3ea797808ef.png)
